tests/ci_build/Dockerfile.ubuntu1404_crosstool

```diff
@@ -0,0 +1,22 @@
+FROM ubuntu
```

Comment:
please consider rename it to `Dockerfile.crosstool`, we use ubuntu 14.04 as the default os